pynecone/route.py

```diff
@@ -0,0 +1,42 @@
+"""The route decorator and associated variables."""
+
+DECORATED_ROUTES = []
+
+
+def route(route=None, title=None, image=None, description=None, on_load=None):
+    """Decorate a function as a page.
+
+    pc.App() will automatically call add_page() for any method decorated with route
+    when App.compile is called.
+
+    Note: the decorated functions still need to be imported
+
+    Args:
+        route (_type_, optional): The route to reach the page. Defaults to None.
```

Comment:
Let's add the type annotations in the function signature and remove them from the docstrings

Reply:
sure